config/locales/en.yml

```diff
@@ -1055,6 +1053,13 @@ en:
         max_res: "Maximum Reservation Minutes"
         cancel_hours: "Minimum Cancelation Hours"
         cancel_mins: "Automatic Cancelation Minutes"
+    offline:
+      note: Instrument is down
+      notice: |
+        This instrument is currently down.
+        Contact facility for more information.
+        It will be canceled for $0 if the instrument is still down at the time
```

Comment:
What would you think of "It will be canceled at no charge if..."? I'm thinking that might read a little nicer from the customer's perspective.